week_1/html1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head class="hdd1">
+<title>My Guest Book</title>
+<link rel="stylesheet" type="text/css" href="css1.css">
+<link href="https://fonts.googleapis.com/css2?family=Kalam&display=swap" rel="stylesheet">
+<div class="tagr"><h1 class="hd">My Guest Book</h1>
+<h3 class="hd1">Please sign My Guest Book.Thanks!!!</h3></div>
```

Comment:
Add this inside the `body`.

Reply:
OK

Reply:
The changes are made, please  verify it.

Reply:
Looks good